assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -91,6 +110,33 @@ describe('Upgrade Landing Controller', function() {
                     assert.isFalse(value.status);
                 });
             });
+
+            it('should contain checks for addons', function () {
+                expect(controller.prechecks.checks).toEqual(
+                    jasmine.objectContaining({
+                        'clusters_healthy': jasmine.objectContaining(
+                            { status: jasmine.any(Boolean), label: jasmine.any(String) }
+                        ),
+                        'ceph_healthy': jasmine.objectContaining(
+                            { status: jasmine.any(Boolean), label: jasmine.any(String) }
```

Comment:
If you want this kind of check here, the whole label can be hardcoded in the expectation but I think the main point of this check is that the keys are as they should.